packages/@lwc/integration-karma/test/template/directive-if-elseif-else/index.spec.js

```diff
@@ -2,6 +2,7 @@ import { createElement } from 'lwc';
 import XComplex from 'x/complex';
 import XTest from 'x/test';
 import XForEach from 'x/forEach';
+import XparentWithNamedSlot from 'x/parentWithNamedSlot';
```

Comment:
```suggestion
import XParentWithNamedSlot from 'x/parentWithNamedSlot';
```

Just following the naming convention of the other classes.